src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/SafeArea.shared.cs

```diff
@@ -51,9 +51,9 @@ bool Equals(SafeArea other)
 			Bottom == other.Bottom);
 
 		public override bool Equals(object obj)
-			=> !ReferenceEquals(null, obj) &&
-			obj is SafeArea safeArea &&
-			Equals(safeArea);
+			=> obj is not null
```

Comment:
Can we convert this `struct` to be a `record`?

Reply:
Records work fine in Mono; they're just C# syntax that get compiled down to existing .NET runtime elements. Heck, you can even use Records with .NET Framework: https://blog.ndepend.com/using-c9-record-and-init-property-in-your-net-framework-4-x-net-standard-and-net-core-projects/

I'll convert this 👍

Reply:
That being said, we probably should leave this as a `struct` because if we change it to a `record`, it may prevent developers who haven't yet upgraded to C# 9.0 from using the CommunityToolkit